charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -79,6 +79,9 @@ spec:
             {{- with .Values.controller.k8sTagClusterId }}
             - --k8s-tag-cluster-id={{ . }}
             {{- end }}
+            {{- if .Values.controller.enableMetrics }}
```

Comment:
This should probably explicitly check the user hasn't manually specified `.Values.controller.httpEndpoint`, or else there will be a conflict

Reply:
Good catch 👍  

```  
 httpEndpoint: "localhost:8080"
 enableMetrics: true
```

with

```
{{- if and (.Values.controller.enableMetrics) (not .Values.controller.httpEndpoint) }}
```

output:

```
    Args:
      controller
      --endpoint=$(CSI_ENDPOINT)
      --http-endpoint=localhost:8080
      --logtostderr
      --v=2
```

--- 
no http endpoint specified for good measure:

```
 httpEndpoint: 
 enableMetrics: true
```

with
```
{{- if and (.Values.controller.enableMetrics) (not .Values.controller.httpEndpoint) }}
```

output:
```
    Args:
      controller
      --endpoint=$(CSI_ENDPOINT)
      --http-endpoint=0.0.0.0:3301
      --logtostderr
      --v=2
```